test/cli/templates/cli-test.js

```diff
@@ -0,0 +1,34 @@
+const { parseOptions } = require('commander');
+const { Command } = require('commander');
+
+/**
+ * @method cliTest
+ * @description returns a new commander instance after parsing the provided arguements
+ * @param {string[]} args Array of arguements to by parsed via commander
+ */
+
+module.exports = (args) => {
+  let program = new Command();
+  program
+    .option('-i, --image [PATH/URL]', 'Input image URL')
+    .option('-s, --step [step-name]', 'Name of the step to be added.')
+    .option('-o, --output [PATH]', 'Directory where output will be stored.')
+    .option('-b, --basic', 'Basic mode outputs only final image')
+    .option('-c, --config [Object]', 'Options for the step')
+    .option(
+      '--save-sequence [string]',
+      'Name space separated with Stringified sequence'
+    )
+    .option(
```

Comment:
Are these the only options? What if they are changed in the future? Can it be dynamically loaded from the sequencer file?

Reply:
I will have to look into it. It seems like the options are manually defined in `index.js`.

Reply:
O:+1:

Reply:
Yes, it does seem like we're almost rebuilding the CLI in order to test the CLI here. Maybe we need `cliTest()` to accept an argument and use it in each test with just `cliTest('--save-sequence [string]')` -- so that we can see the scope of each test? Or something like that...